core/nodejsActionBase/launcher.js

```diff
@@ -27,9 +29,16 @@ try {
     } catch (e) {}
   })();
 
-  const lambdaCompat = process.env.__OW_LAMBDA_COMPAT === undefined ? false : process.env.__OW_LAMBDA_COMPAT.toLowerCase() === 'true' && NodeActionLambdaRunner !== undefined;
   const handler = eval('require("./##MAIN_FILE##").##MAIN_FUNC##') // Will be replaced in the compile script with the correct main.
-  const runner = lambdaCompat === false ? new NodeActionRunner(handler) : new NodeActionLambdaRunner(handler);
+
+  // Deem function to be Lambda if env var is provided (for backwards
+  // compatibility) or if the function signature is detected to be one for
+  // Lambda.
+  const lambdaCompat = ((process.env.__OW_LAMBDA_COMPAT !== undefined && process.env.__OW_LAMBDA_COMPAT.toLowerCase() === 'true')
+      && NodeActionLambdaRunner !== undefined)
```

Comment:
nit: This is never going to be false for our runtimes. Should we drop the `try/catch` and assume it's available always?

Reply:
I'm not comfortable dropping the try/catch yet (I just don't have enough experience with this code base to know exactly what it could catch), but I do agree I can get rid of the check for whether the LambdaRunner was `require`'d successfully. We control this code. It'll always be there. I'll change that.

Reply:
That's the try/catch I meant, fwiw.